scripts/mail-alarm

```diff
@@ -480,6 +491,46 @@ class HAHostFailedETG(EmailTextGenerator):
 
         return body_pattern % self.text
 
+class SRIOThroughputTotalAlertETG(EmailTextGenerator):
+  def __init__(self, cls, obj_uuid, value, alarm_trigger_period, alarm_trigger_level, sr_uuid, mail_language):
+        super(SRIOThroughputTotalAlertETG, self).__init__(mail_language, 'sr_io_throughput_total_alert')
+        if alarm_trigger_period is None:
+            alarm_trigger_period = 60
+        if cls == 'Host':
+            self.params = get_host_params(obj_uuid)
+        else:
+            raise Exception, "programmer error"
```

Comment:
This error is too generic, please add something like ` - this alarm should only be available for Hosts"` or something like that